packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -184,20 +233,27 @@ class MessageList extends Widget<Properties> {
     items.forEach((item, index) => {
       const newMessageGroupItem = item ?? {};
       const id = newMessageGroupItem.author?.id;
-
+      const shouldCreateDayHeader = this._shouldAddDayHeader(item?.timestamp);
       const isTimeoutExceeded = this._isTimeoutExceeded(
         currentMessageGroupItems[currentMessageGroupItems.length - 1] ?? {},
         item,
       );
+      const shouldCreateMessageGroup = (shouldCreateDayHeader && currentMessageGroupItems.length)
```

Comment:
If i remove the following part '(shouldCreateDayHeader && currentMessageGroupItems.length) ||' no any failed tests. Kindly check it and add tests scenarios.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925